dolphinscheduler-server/src/test/java/org/apache/dolphinscheduler/server/worker/task/datax/DataxTaskTest.java

```diff
@@ -0,0 +1,345 @@
+
+        String fileName = String.format("%s/%s_node.sh", props.getTaskDir(), props.getTaskAppId());
+        Mockito.when(shellCommandExecutor.run(fileName, processDao)).thenReturn(0);
+    }
+
+    private DataSource getDataSource() {
+        DataSource dataSource = new DataSource();
+        dataSource.setType(DbType.MYSQL);
+        dataSource.setConnectionParams(
+                "{\"user\":\"root\",\"password\":\"123456\",\"address\":\"jdbc:mysql://127.0.0.1:3306\",\"database\":\"test\",\"jdbcUrl\":\"jdbc:mysql://127.0.0.1:3306/test\"}");
+        dataSource.setUserId(1);
+        return dataSource;
+    }
+
+    private ProcessInstance getProcessInstance() {
+        ProcessInstance processInstance = new ProcessInstance();
+        processInstance.setCommandType(CommandType.START_PROCESS);
+        processInstance.setScheduleTime(new Date());
+        return processInstance;
+    }
+
+    @After
+    public void after()
+        throws Exception {}
+
+    /**
+     * Method: DataxTask()
+     */
+    @Test
+    public void testDataxTask()
+            throws Exception {
+        TaskProps props = new TaskProps();
+        props.setTaskDir("/tmp");
+        props.setTaskAppId(String.valueOf(System.currentTimeMillis()));
+        props.setTaskInstId(1);
+        props.setTenantCode("1");
+        dataxTask = new DataxTask(props, logger);
+    }
+
+    /**
+     * Method: init
+     */
+    @Test
+    public void testInit()
+            throws Exception {
+        dataxTask.init();
+    }
+
+    /**
+     * Method: handle()
+     */
+    @Test
+    public void testHandle()
+            throws Exception {
+        try {
+            dataxTask.handle();
+        } catch (RuntimeException e) {
+            if (e.getMessage().indexOf("process error . exitCode is :  -1") < 0) {
+                Assert.fail();
+            }
+        }
+    }
+
+    /**
+     * Method: cancelApplication()
+     */
+    @Test
+    public void testCancelApplication()
+            throws Exception {
+        dataxTask.cancelApplication(true);
+    }
+
+    /**
+     * Method: parsingSqlColumnNames(DbType dsType, DbType dtType, BaseDataSource
+     * dataSourceCfg, String sql)
+     */
+    @Test
+    public void testParsingSqlColumnNames()
+        throws Exception {
+        try {
+
+            BaseDataSource dataSource = DataSourceFactory.getDatasource(getDataSource().getType(),
+                    getDataSource().getConnectionParams());
+
+            Method method = DataxTask.class.getDeclaredMethod("parsingSqlColumnNames", DbType.class, DbType.class, BaseDataSource.class, String.class);
+            method.setAccessible(true);
+            String[] columns = (String[]) method.invoke(dataxTask, DbType.MYSQL, DbType.MYSQL, dataSource, "select 1 as a, 2 as `table` from dual");
+
+            Assert.assertNotNull(columns);
+
+            Assert.assertTrue(columns.length == 2);
+
+            Assert.assertEquals(Arrays.toString(columns), "[`a`, `table`]");
```

Comment:
assertEquals first param is expected value, sceond param is actual value, please swap there params.